src/vue/components/panel-compact.vue

```diff
@@ -124,59 +137,70 @@
              v-html="data.fullDefinitions"></div>
       </div>
       <div :id="inflectionsPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflections"
-           v-if="data.inflectionComponentData.inflDataReady && data.settings" v-show="inflectionsTabVisible">
-        <inflections :data="data.inflectionComponentData" :inflections-enabled="data.inflectionsEnabled"
-                     :locale="data.settings.locale.currentValue" :wait-state="data.inflectionsWaitState"
-                     @contentwidth="setContentWidth" class="alpheios-panel-inflections">
-        </inflections>
+           v-if="$store.getters[`app/hasInflData`]" v-show="inflectionsTabVisible">
+        <inflections @contentwidth="setContentWidth" class="alpheios-panel-inflections"></inflections>
       </div>
       <div :id="inflectionsBrowserPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflectionsbrowser"
-           v-if="data.inflectionBrowserEnabled && data.settings" v-show="inflectionsBrowserTabVisible">
-        <inflection-browser :data="data.inflectionBrowserData" :infl-browser-tables-collapsed="data.inflBrowserTablesCollapsed"
+           v-if="settings.contentOptions.items" v-show="$store.state.app.tabState.inflectionsbrowser">
+        <inflection-browser :data="data.inflectionBrowserData"
+                            :infl-browser-tables-collapsed="data.inflBrowserTablesCollapsed"
```

Comment:
I'm not sure because I'm not fully familiar with this logic: when there are several people doing changes (or myself a long time ago 🙂) it's hard to be up to date. I will examine it to see what we use it for.

Reply:
I am lost here too :-)

Reply:
OK, I think I now remember we used it when having both regular inflection tables and tables of the browser on the same page. So when regular tables were expanded, browser's tables needs to be collapsed and vice versa. Since the two are now on different tabs, we don't need it anymore, I assume.